src/tree.rs

```diff
@@ -582,14 +582,28 @@ impl<Pane> Tree<Pane> {
         &mut self,
         remove_me: TileId,
     ) -> Option<(TileId, usize)> {
+        let mut result = None;
+
         for (parent_id, parent) in self.tiles.iter_mut() {
             if let Tile::Container(container) = parent {
                 if let Some(child_index) = container.remove_child(remove_me) {
-                    return Some((*parent_id, child_index));
+                    result = Some((*parent_id, child_index));
                 }
             }
         }
-        None
+
+        // Make sure that if we drag away the active some tabs,
+        // that the tab container gets assigned another active tab.
+        if let Some((parent_id, _)) = result {
+            if let Some(mut tile) = self.tiles.remove(parent_id) {
+                if let Tile::Container(Container::Tabs(tabs)) = &mut tile {
+                    tabs.ensure_active(&self.tiles);
+                }
+                self.tiles.insert(parent_id, tile);
```

Comment:
Considering the case where "Tab Container" has two children A and B, with B active. Drag B to the same "Tab Container". Wouldn't that code cause A to become active?

Reply:
Good question! But a the moved tab always becomes the active tab wherever it ends up

Reply:
I just tried, and failed, to demonstrate that I'm saying is actually happening, so it's probably ok 🤷🏻

Reply:
I don't know how to parse that sentence

Reply:
I built rerun with that PR and tried to see if I could obtain the faulty behaviour of my original comment. I couldn't (aka it's still working as expected). So I suppose that nullify my original comment.